hat/assets/js/apps/Iaso/domains/users/hooks/useGetCurrentUser.ts

```diff
@@ -0,0 +1,36 @@
+import { UseQueryResult } from 'react-query';
+import { useDispatch } from 'react-redux';
+import { useSnackQuery } from '../../../libs/apiHooks';
+import { getRequest } from '../../../libs/Api';
+import { Profile } from '../../teams/types/profile';
+import { setCurrentUser } from '../actions';
+
+const getCurrentUser = (): Promise<Profile> => {
+    return getRequest('/api/profiles/me/');
+};
+export const useGetCurrentUser = (
+    enabled: boolean,
+): UseQueryResult<Profile, Error> => {
+    const queryKey: any[] = ['currentUser'];
+    const dispatch = useDispatch();
+    // @ts-ignore
+    return useSnackQuery({
+        queryKey,
+        queryFn: () => getCurrentUser(),
+        dispatchOnError: false,
```

Comment:
Why not dispatch on Error for this one? I don't think we have any front-end that manages this error